styles2.css

```diff
@@ -0,0 +1,662 @@
+
+*{
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+a{
+    text-decoration: none;
+}
+
+h1,h2,h3 {
+    font-family: 'Merriweather', serif;
+}
+
+p,button{
+    font-family: 'Roboto', sans-serif;;
+}
+
+.outer-container{
+    color: white;
+    display:flex;
+    flex-direction:column;
+    align-items: center;
+    text-align: center;
+    height: 650px;
+    background: #5722CD;
+    background: linear-gradient(90deg, #5722CD 0%,  #4845E4 100%);
+}
+
+.navbar ul {
+    display: flex;
+    flex-direction:row;
+    gap: 60px;
+    justify-content: center;
+    margin: 30px;
+
+}
+
+.navbar li {
+    list-style: none;
+}
+
+.navbar a {
+    text-decoration: none;
+    color: white;
+    font-weight: bold;
+}
+
+.hero-section h1 {
+margin: 30px;
+}
+.hero-section p{
+    margin: 30px;
+}
+.hero-section button{
+    margin: 30px;
+    padding: 15px;
+    border-radius: 45px;
+    background-color: #ED4385;
+    color: white;
+    width: 200px;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.10), 0 6px 20px 0 rgba(0,0,0,0.40);
+}
+
+
+.swirl {
+    display: flex;
+    position: relative;
+    bottom: -145px; }
+
+
+
+.skewed{
+    height: 100px;
+    width: 100%;
+    color: white;
+    background-color: white;
+    position: relative;
+    bottom: -135px;
+    transform: rotate(2deg);
+}
+
+
+.main1{display: flex;
+    margin-top:140px;
+    flex-direction:column;
+    justify-content:center;
+    align-items: center;
+    text-align:center;
+    color: #5722CD;
+    font-weight: bold;
+}
+
+
+
+.question-section{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+
+}
+
+.question-tekst{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin: 20px;
+    color: black;
+    font-weight: bold;
+    font-size: small;
+
+}
+
+.section-photo1{
+    display: flex;
+    flex-direction: column;
+    width: 500px;
+    margin-top: 200px;
+}
+
+
+.laptop-image{
+    display: flex;
+    justify-content: center;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.10), 0 6px 20px 0 rgba(0,0,0,0.40);
+}
+
+.text-section-photo1{
+    display: flex;
+    flex-direction: column ;
+    text-align: left;
+    margin-top: 10px;
+    gap: 30px;
+}
+
+.squiggle-line{
+    display: flex;
+    justify-content: flex-start;
+    align-content: flex-start;
+    align-items: flex-start;
+    margin-bottom: 30px;
+    width: 50px;
+}
+
+#rolling-stone-text{
+    color: black;
+    font-weight: bold;
+}
+#rolling-stone-title{
+    color: black;
+    font-weight: bold;
+}
+
+
+.section-photo2{
+    display: flex;
+    flex-direction: column;
+    width: 500px;
+    margin-top: 20px;
+
+}
+.books-table{
+    display: flex;
+    justify-content: center;
+    margin-top: 20px;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.10), 0 6px 20px 0 rgba(0,0,0,0.40);
+}
+
+.text-section-photo2{
+    display: flex;
+    flex-direction: column ;
+    text-align: left;
+    gap: 30px;
+    margin-top: 10px;
+}
+
+#variety-text{
```

Comment:
Let goed op of je niet te vaak in herhaling valt: wat kan je een gelijke class geven en wat niet, wat kan je allemaal in een class selector bij elkaar laten vallen en wat niet. Dit zou de code ook een stuk beter leesbaar maken!